### ISSUE TITLE
The training set score is high, but the verification set score is low

### ISSUE THREAD

**Issue by clxa** [layumi/Person_reID_baseline_pytorch#234]
My training log is as follows, why is the training set far lower than the verification set score
Epoch 0/19
----------
/usr/local/lib/python3.6/dist-packages/torch/optim/lr_scheduler.py:123: UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
  "https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate", UserWarning)
train Loss: 11.4538 Acc: 0.0000
val Loss: 10.8049 Acc: 0.2386
Training complete in 13m 54s

Epoch 1/19
----------
train Loss: 10.5938 Acc: 0.0072
val Loss: 8.7350 Acc: 0.5530
Training complete in 27m 44s

Epoch 2/19
----------
train Loss: 8.6462 Acc: 0.0213
val Loss: 4.5044 Acc: 0.7199
Training complete in 41m 31s

**Comment by layumi** [comment]
Hi @clxa 
Do you use PCB or random erasing? 
If the training loss decreases, the model usually is OK... Take it easy.
